test/helpers/tickets_helper_test.rb

```diff
@@ -1,7 +1,8 @@
 class TicketsHelperTest < ActionView::TestCase
   test "status boolean to word" do
-    assert_equal status_word(true), "Approved"
-    assert_equal status_word(false), "Pending"
+    assert_equal status_word(:approved), "Approved"
```

Comment:
Needs to be changed as per the changes in the method itself. maybe a mock ticket can be made for better testing, since this method should always be used on a `ticket.status`, not the symbols themselves